DNSChecker/getAuthoritativeServers.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+#DNSHEALTH-12
+import dns.query, dns.resolver
+from dns.exception import DNSException
+
+# a functions that returns all the authoritative servers for the domain given.
+# Takes a domain as an argument and returns a tuple of the OK to indicate that it succeed 
+# and a list of tuple with all the authoritative domains.
+def getAuthoritativeServers(domain):
+
+    defaultResolver = dns.resolver.get_default_resolver()
+    
+    nameServers = defaultResolver.nameservers[0]
+
+    arrayOfDomains = domain.split('.')
```

Comment:
Wrong naming of variables: 
Python's way is var_1. In your case defultResolver, shoud be default_resolver and so on. Refactor everywhere where needed